resources/js/Pages/Study/About.vue

```diff
@@ -682,6 +693,10 @@ export default {
             if (this.smiles && this.smiles != "") {
                 this.editor.setSmiles(this.smiles);
             }
+
+            if (this.editor.getSmiles() == "") {
+                this.errorMessage = "The entered SMILES is not valid.";
+            }
```

Comment:
Use OCL molecule module to read smiles and generate molecule. If there are any errors, then display the appropriate message.

Reply:
let mol = OCL.molecule.fromSmiles("CCCC")

https://cheminfo.github.io/openchemlib-js/classes/Molecule.html#fromSmiles